Rakefile

```diff
@@ -4,11 +4,32 @@
 # and started before any application code is loaded.
 require 'simplecov' if ENV['COVERAGE']
 
-desc 'Check for no pending changelog entries before release'
-task release: 'changelog:check_clean' # Before task is required
```

Comment:
I believe removing this would allow a release to be published without the changelog being updated. Doesn't look like you added that back anywhere. There are always a few commits happening around a release:
* https://github.com/rubocop/rubocop/commit/fc7022ab9aa0a84a7d9bafa9e2612ea0b1b08092
* https://github.com/rubocop/rubocop/commit/05f0b56b2f6c433e62db2f00a2b723fb65c04f84
* https://github.com/rubocop/rubocop/commit/f44b15c0ccbd60a82d5bad820662cbc532917e1a

I'm not sure on the process for this, a maintainer would be able to answer better

Reply:
I could certainly use some help around the `release` task to ensure I'm not breaking anything so feedback or changes here (feel free to yeet commits to this PR) would be great.